data/favicon.json

```
@@ -128,6 +128,7 @@
 	"ikea.com": "ikea.com.ico",
 	"ikeafamily.no": "ikea.com.ico",
 	"iliad.it": "iliad.it.png",
+	"instantsystems.se: "instantsystems.se.png",
```

you forgot the closing `"`
```suggestion
	"instantsystems.se": "instantsystems.se.png",
```

---

Hi there
Oh, totally that!
Thanks for pointing out and solving it aswell :)